OrthoEvol/Orthologs/Blast/comparative_genetics.py

```diff
@@ -73,34 +78,62 @@ def __init__(self, project=None, project_path=os.getcwd(), acc_file=None,
         self.__pre_blast = pre_blast
         self.__post_blast = post_blast
         self.__taxon_filename = taxon_file
+        self.acc_file = acc_file
         self.acc_csv_filename = acc_file
+
+        # Initialize variables
+        self.project_path = project_path
         self.project = project
+        self.species = 'Homo_sapiens'
```

Comment:
If it's going to be a class variable, then go ahead and turn it into a parameter.  Maybe call it a `self.reference_species`?

Reply:
I mostly add this here for the ability to use across the inherited classes instead of having to explicitly use that all across the code for exactly your suggestion here...just wasn't sure if we wanted to go ahead and roll that out - yet, but I'd be happy to add that.